src/widget.tsx

```diff
@@ -119,11 +124,29 @@ export class CommentWidget<T> extends ReactWidget {
       );
     };
 
+    // This should be able to work without the bind?
     const CommentWrapper = _CommentWrapper.bind(this);
 
     return <CommentWrapper comment={this.comment!} />;
   }
 
+  protected _deleteReply(rcomment: IComment): void {
+    const comments = this._metadata.get('comments');
+    const commentList = comments as any as IComment[];
+    const commentIndex = commentList.findIndex(c => c.id === this.commentID);
+    const comment = commentList[commentIndex];
+    if (rcomment != null) {
```

Comment:
rcomment will never be null since it's type is `IComment` and it's not an optional parameter